sys/fuchsia/fidl_io.txt

```diff
@@ -52,24 +119,35 @@ fuchsia_io_ObjectDescribeRequestHandles {
 	void	void
 } [packed]
 
-fuchsia_io_ObjectOnOpenRequest {
-	hdr	fidl_message_header[2147483655]
+fuchsia_io_ObjectDescribeResponse {
+	hdr		fidl_message_header[2147483654]
+	infoInLine	fidl_union[fuchsia_io_ObjectInfoInLine]
```

Comment:
If there is out-of-line data for a union, where will it go? Do we have any examples here?

Reply:
I do not have examples yet. As far as I understand, out-of-line data (if present) should be the next out-of-line object.